### PR TITLE
Add the error response and the logger

### PR DESCRIPTION
**Pull request by hyunsikjeong**

### CODE REVIEW

File: proxy.py [hunk 1]
```diff
@@ -14,6 +16,10 @@
                         help='binding port')
     parser.add_argument('--forward', type=int, default=8080,
                         help='port to forward (deafult: 8080)')
+    parser.add_argument('--log', default='proxy.log',
+                        help='log file path (default: \"proxy.log\")')
+    parser.add_argument('--debug', action='store_true',
+                        help='debug mode (default: False)')
```
Comment (sgkim126) [inline proxy.py]:
```python
help='running in debug mode')
```
The current description seems that `--debug` flag takes a boolean value.

**Comment by sgkim126** [comment]
@joojis Would you check the first commit? Is the format that sdk expects?

File: proxy.py [hunk 2]
```diff
@@ -22,20 +28,31 @@
...
+    app.logger.addHandler(handler)
+    app.logger.setLevel(logging.INFO)
+
     @app.route('/', methods = ['POST'])
     def proxy():
-        if not request.is_json: # Invalid JSON Format
-            abort(404)
+        if not request.is_json: # Content type is not json 
+            app.logger.info('Received the data which is not JSON [{}]'.format(request.remote_addr))
+            abort(400)
```
Comment (sgkim126) [inline proxy.py]: Why do you change 404 Not Found to 400 Bad Request?

**Review by sgkim126 — CHANGES_REQUESTED** [review]

File: proxy.py (hunk 2 above)
Comment (hyunsikjeong) [reply proxy.py]: Read the commit message. [Link](https://github.com/CodeChain-io/codechain-proxy/pull/3/commits/2d7d172c9e521325b9765a85382a8c81747ba783)

Comment (hyunsikjeong) [reply proxy.py]: Divided into two commits.

**Review by sgkim126 — APPROVED** [review]

**Comment by sgkim126** [comment]
LGTM. Let's merge it when @joojis approves it.

**Review by joojis — APPROVED** [review]
Looks good to me.

**Pull request merged**